gen/injection.js

```diff
@@ -0,0 +1,166 @@
+aiDataGraphqlPost = () => {
+    //INIT_INJECTION
+    if (!query || !variables) {
+        sys.logs.error('Invalid argument received. This helper should receive the following parameters as non-empty strings: [query,variables].');
+        return;
+    }
+    var url = parse('/v3/graphql');
+    sys.logs.debug('[veritone] POST from: ' + url);
+    httpOptions = httpOptions || {};
```

Comment:
I think there is a missing 'var' declaration for some variables.

Reply:
In the injection file there is no need for these declarations, since it is a code that is copied exactly the same in the functions.js file. In the place where this code is injected it is defined as an argument of the original function.